pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["jupyter_packaging~=0.9,<2"]
+requires = ["jupyter_packaging~=0.9"]
```

Comment:
I think we want this to be `>=0.10,<2` since we are compatible with the planned 1.0 interface for `Jupyter-packaging`

Reply:
I can do that if you want, just didn't want to change the current behaviour without a jupyter expert to tell me to

Reply:
Does that mean this is no longer compatible with 0.9?

Reply:
It is, this just sets a minimum on what will be used in the build environment

Reply:
Hmm, I'm comfortable with changing the upper bound since 1.X hasn't been released yet anyway, but I'm not sure if I'm comfortable with changing the lower bound. If this currently builds with 0.9, I see no reason to increase the lower bound.

Reply:
Fair.  :smile: